code/__defines/chemistry.dm

```diff
@@ -11,7 +11,6 @@
 
 #define SOLID 1
 #define LIQUID 2
-#define GAS 3
```

Comment:
although not currently used, please retain this, it will be used in future

maybe add in a dummy chemical that uses it to prevent dreamchecker from complaining

Reply:
Duplicate Define. I checked everything and can say that GAS don't used. If you planned this define add in code - you can add in feature.